src/controlnet_aux/hed/__init__.py

```diff
@@ -78,7 +78,7 @@ def to(self, device):
         self.netNetwork.to(device)
         return self
     
-    def __call__(self, input_image, detect_resolution=512, image_resolution=512, safe=False, return_pil=True, scribble=False):
+    def __call__(self, input_image, safe=False, scribble=False, detect_resolution=512, image_resolution=512, return_pil=True):
```

Comment:
Is this really necessary? This would break the positional ordering

Reply:
Sure, there's something for consistency but yeah fine to keep as-is